releasenotes/notes/security-agent-host-a291030fdebf8464.yaml

```
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    Do not activate security-agent service by default on host install (was already properly starting and exiting if not activated in configuration)
```

`host install` is a bit of an internal name, we could use instead:

```suggestion
    Do not activate security-agent service by default in the Linux packages of the Agent (RPM/DEB). The security-agent was already properly starting and exiting if not activated in configuration.
```